src/server/biosecurity/buildings-how-minimise-contamination/index.js

```diff
@@ -0,0 +1,13 @@
+import { Page } from '../../common/model/page/page-model.js'
+
+// using Page to not make whole section invalid due to the page not being implemented yet
+// make QuestionPage when implementing
+export class BuildingsHowMinimiseContaminationPage extends Page {
+  urlPath = '/biosecurity/buildings-how-minimise-contamination'
+  sectionKey = 'biosecurity'
+  question = 'question to be asked'
+  questionKey = 'buildingsHowMinimiseContamination'
```

Comment:
I don't think we need a question / questionKey here, FWIW?  Until it's a QuestionPage.

Not a blocker to merging

Reply:
Maybe not strictly required but as we are building them one after the other this week I think it is fine as it is